k8_cortx_cloud/rolling-upgrade-cortx-cloud.sh

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env bash
```

Comment:
Can we name as upgrade.sh or upgrade-cortx-cloud.sh ? By default upgrade must be rolling upgrade so no need to qualify it explicitly.

Reply:
there is already one script in k8s code as upgrade-cortx-cloud.sh which is for disruptive upgrade.

Reply:
yes Working on it.

Reply:
done